examples/example.py

```diff
@@ -2,17 +2,21 @@
 from quartz_solar_forecast.forecast import run_forecast
 from quartz_solar_forecast.pydantic_models import PVSite
 from datetime import datetime, timedelta
+import asyncio
 
-def main():
+async def main():
```

Comment:
Why does the example with no inverter used need to be async?

Reply:
> Why does the example with no inverter used need to be async?

This is because it calls the async run_forecast function, and it throws some loc errors if any async function is called synchronously

Reply:
As mentioned `asyncio.run` might be a good way to get around this